src/META

```diff
@@ -1,7 +1,7 @@
 version="1.2.0"
 description="Code coverage for OCaml"
 requires="bisect_ppx.runtime"
-ppx="./bisect_ppx"
+ppx="./bisect_ppx --as-ppx"
```

Comment:
The META should contain various additional things to play nicely with the omp driver. There are exemples in omp's repository.